webpack/webpack.dev.js

```diff
@@ -73,6 +74,17 @@ const devConfig = {
     ],
     module: {
         rules: [
+            {
```

Comment:
removing `ExtractTextPlugin` from dev config as `react-hot-load` takes care of hot loading css modules so long as the `ExtractTextPlugin` isn't applied to our css loaders.